### PR TITLE
RI-4852 - Use msgpackr alternative library to encode/decode msgpack contents

### PR DESCRIPTION
**Pull request by GnaneshKunal**
Fixes https://github.com/RedisInsight/RedisInsight/issues/2391

Test data:
```
echo '98A44E6F6E6581A15499CD07E7CCE9090408CE26DFE89700000081A15499CD07E7CCE9090409CE05B137F3000000B26C73612F7069755F61737369676E5F7266738192AD637573746F6D65725F6E616D65C081AA556E7265736F6C766564A87B2E2F6E616D657D8081A573746570739080' | xxd -r -p | redis-cli -x SET msg-pack-complex-2
```

Before:
image

After:
image

### CODE REVIEW

**Comment by mariasergeenko** [comment]
qa approve

**Review by AmirAllayarovSofteq — DISMISSED** [review]

File: redisinsight/ui/src/utils/formatters/valueFormatters.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { decode, encode } from '@msgpack/msgpack'
+import { encode, decode } from 'msgpackr';
```
Comment (AmirAllayarovSofteq) [inline redisinsight/ui/src/utils/formatters/valueFormatters.tsx:1]: Remove semicolon, please

File: redisinsight/ui/src/utils/formatters/valueFormatters.tsx [hunk 2]
```diff
@@ -72,7 +72,7 @@ const formattingBuffer = (
     case KeyValueFormat.JSON: return bufferToJSON(reply, props as FormattingProps)
     case KeyValueFormat.Msgpack: {
       try {
-        const decoded = decode(reply.data)
+        const decoded = decode(Uint8Array.from(reply.data));
```
Comment (AmirAllayarovSofteq) [inline redisinsight/ui/src/utils/formatters/valueFormatters.tsx:75]: And here, too

File: redisinsight/ui/src/utils/formatters/valueFormatters.tsx (hunk 1 above)
Comment (GnaneshKunal) [reply redisinsight/ui/src/utils/formatters/valueFormatters.tsx]: Done

File: redisinsight/ui/src/utils/formatters/valueFormatters.tsx (hunk 2 above)
Comment (GnaneshKunal) [reply redisinsight/ui/src/utils/formatters/valueFormatters.tsx]: Done

**Review by romansergeenkosofteq — APPROVED** [review]

**Pull request merged**